2nd-gen/packages/swc/components/status-light/stories/status-light.stories.ts

```diff
@@ -47,40 +41,85 @@ argTypes.size = {
     options: StatusLight.VALID_SIZES,
 };
 
-args['default-slot'] = 'Status light';
-args.size = 'm';
-
 /**
- * An `<sp-status-light>` is a great way to convey semantic meaning, such as statuses and categories.
- * It provides visual indicators through colored dots accompanied by descriptive text.
+ * Status lights describe the condition of an entity. Much like [badges](../?path=/docs/components-badge--readme), they can be used to convey semantic meaning, such as statuses and categories.
  */
 const meta: Meta = {
     title: 'Status light',
     component: 'swc-status-light',
-    argTypes,
     parameters: {
         docs: {
             subtitle: `Status lights convey semantic meaning through colored dots accompanied by descriptive text.`,
         },
+        design: {
+            type: 'figma',
+            url: 'https://www.figma.com/design/Mngz9H7WZLbrCvGQf3GnsY/S2---Desktop?node-id=36797-954',
+        },
+        stackblitz: {
+            url: 'https://stackblitz.com/edit/vitejs-vite-y2kz1rkx?file=package.json',
+        },
+        flexLayout: 'row-wrap',
     },
     args,
+    argTypes,
     render: (args) => template(args),
     tags: ['migrated'],
 };
 
 export default meta;
 
 // ────────────────────
-//    AUTODOCS STORY
+//    HELPERS
 // ────────────────────
 
-type StatusLightVariant = typeof StatusLight.prototype.variant;
-type StatusLightSize = typeof StatusLight.prototype.size;
+const semanticLabels = {
```

Comment:
this is the pattern for custom labeling when using type consts to generate templates in a story

Reply:
Very nice